packages/google-tag-manager/index.js

```diff
@@ -7,12 +7,13 @@ module.exports = async function nuxtTagManager(_options) {
     layer: 'dataLayer',
     pageTracking: false,
     respectDoNotTrack: false,
+    dev: false,
```

Comment:
I'm not sure about this but like analytics, we can make it enabled by default and release a breaking change.

Reply:
I'm fine with it!